pythonitalia/pycon
# Send CFP notifications via SQS

## Why


We want to send slack notifications when someone sends a CFP

## How to test

- Go to https://pycon-gvxzz6zpg-python-italia.vercel.app/en/cfp
- Create an account on staging
- Submit a CFP!
- Go to slack and open the channel website-test

You should see your submission there

<img width="1110" alt="image" src="https://user-images.githubusercontent.com/3382153/142082435-33e67a23-b75b-47ca-8952-319bc0340e3d.png">
